test/Datadog.Trace.Tests/TracerTests.cs

```diff
@@ -402,14 +402,18 @@ public void DoesNotThrowOnCrossDomainCallsWhenLeaseExpired()
             // Assert
             // Nothing. We should just throw no exceptions here
 
-            void CallFromRemote() => Thread.Sleep(200);
+            // Ensure the remote call takes long enough for the lease manager poll to occur.
+            // Even though we reset LifetimeServices.LeaseManagerPollTime to a shorter duration,
+            // the default value is 10 seconds so the first poll may not be affected by our modification
+            void CallFromRemote() => Thread.Sleep(TimeSpan.FromSeconds(12));
```

Comment:
I find a local method like this hard to read. Thoughts on using a static member method or inlining the lambda where it's used above?

Reply:
Yeah that makes sense, I'll revisit this